context.md

```diff
@@ -0,0 +1,47 @@
+# (PART) Context {-}
+
+# The World {#world}
+
+The world is a fork of Tolkien's **Middle Earth**. Set in the **4th Age**, `xxx` centuries after the rule of King Elessar in Gondor.
+
+![](https://i.imgur.com/xRPrU40.png)
+
+## Darkness evil {#evil}
+
+People believe there is pure evil that lives in the depths of the world.
+
+## Main empires
+
+The main empires of the world are descendent kingdoms from the late 3rd Age of Middle Earth.
+The elves have mostly left to the undying lands.
+
+# Sea of Rhûn {#seaofrhun}
+
+The Sea of [Rhûn](https://lotr.fandom.com/wiki/Sea_of_Rh%C3%BBn) was located in the west of the land of Rhûn, which was east of Rhovanion and north of Mordor.
+The sea also contained a wooded island within the Sea.
+
+There were unnamed highlands on the southwest side of the sea and a beech-forest known as [Neldoreth](https://lotr.fandom.com/wiki/Neldoreth "Neldoreth") on the northeast side.
+Northwest of the sea was [Dorwinion](https://lotr.fandom.com/wiki/Dorwinion "Dorwinion"), a land of great gardens, vast vineyards and renowned wines.
+The [Kine of Araw](https://lotr.fandom.com/wiki/Kine_of_Araw "Kine of Araw"), wild, white cattle, were known to live near the sea's shores.
+
+![](https://api2.inkarnate.com/api/embedScenes/oy9y3X/previewImage)
+
+Six major geographical features:
+
+- Highlands (unnamed)
+- Neldoreth (forest)
+- Sea of Rhûn
+- Woden Island (megaplex - ancient structure)
+- Gardens of Dorwinion
+- Swamp in south of Sea? (Chosen from WWN 122)
+
+Six major nations or groups `roll 2 thematic elements WWN 126 for each`
```

Comment:
@brunoric roll once on the table for each of these nations. Then I roll once two, and we mix the results to create a theme for each.